spec/lib/parse_spec.rb

```diff
@@ -486,6 +486,101 @@
     )
   end
 
+  it 'correctly finds expressions and function calls in a CREATE INDEX statment' do
+    query = described_class.parse('CREATE INDEX testidx ON test USING btree (a, (b+c), lower(d))')
+    expect(query.warnings).to eq []
+    expect(query.tables).to eq ['test']
+    expect(query.ddl_tables).to eq ['test']
+    expect(query.call_functions).to eq ['lower']
+    expect(query.tree.stmts.first).to eq(
```

Comment:
It doesn't seem like it's worth it to have this expectation – it makes the test file longer without adding much value

Reply:
I don't disagree, but I copied this pattern from other tests. Thoughts, @lfittl?

Reply:
Yeah, I think its fair to skip this check - its just in a few others to have some coverage of the actual tree structure, but I think its not required here.